### DISCUSSION TITLE
Unsupported file types : .doc

### DISCUSSION THREAD

**Discussion by jogamned** [danny-avila/LibreChat#4661] [Troubleshooting]
### What happened?

Hello,

I'm trying to set which files are accepted by Librechat.

Without any specific configuration, some files are accepted, some are not. Some examples : 
Accepted : images, .json, .pdf, .docx, .txt, .pptx
Not accepted : .sh, .doc

So I've tried the "fileConfig" option in the librechat.yml
``` yaml   
fileConfig:
  endpoints:
    default:
      supportedMimeTypes:
        - "image/.*"
        - "text/.*"
        - "application/.*"
```    

I've also tried to define application/msword, but same result.
According to the logs the config seems to be read correctly here

I've also tried different .doc, I can't attach one here but all should fail I guess

I'm using version 0.7.5 on a GKE cluster.

Thanks for your great work btw, I love this product :)

### Steps to Reproduce

1. Run LibreChat
2. Upload a .doc file

### What browsers are you seeing the problem on?

Chrome

### Relevant log output

```shell
librechat-api-75d85cd647-t5khv {
librechat-api-75d85cd647-t5khv   version: "1.0",
librechat-api-75d85cd647-t5khv   // 2 includedTool(s)
librechat-api-75d85cd647-t5khv   includedTools: ["google","biggiegeturl"],
librechat-api-75d85cd647-t5khv     assistants.disableBuilder: true,
librechat-api-75d85cd647-t5khv     assistants.pollIntervalMs: 750,
librechat-api-75d85cd647-t5khv     assistants.timeoutMs: 180000,
librechat-api-75d85cd647-t5khv     // 7 supportedId(s)
librechat-api-75d85cd647-t5khv     assistants.supportedIds: ["asst_cAGONNPgZXH2zBisXfboACFI","asst_PGd92EaICKh5AC5tF7DnHYyd","asst_e2xM3XMi7OV6671TL8O5huKp","asst_alBCvgpUqr9EzOVftCBlO
librechat-api-75d85cd647-t5khv     // 3 supportedMimeType(s)
librechat-api-75d85cd647-t5khv     default.supportedMimeTypes: ["image/.*","text/.*","application/.*"],

Error: Unsupported file type: application/msword
librechat-api-75d85cd647-24hrp     at fileFilter (/app/api/server/routes/files/multer.js:39:15)
librechat-api-75d85cd647-24hrp     at wrappedFileFilter (/app/node_modules/multer/index.js:44:7)
librechat-api-75d85cd647-24hrp     at Multipart.<anonymous> (/app/node_modules/multer/lib/make-middleware.js:107:7)
librechat-api-75d85cd647-24hrp     at Multipart.emit (node:events:519:28)
librechat-api-75d85cd647-24hrp     at HeaderParser.cb (/app/node_modules/busboy/lib/types/multipart.js:358:14)
librechat-api-75d85cd647-24hrp     at HeaderParser.push (/app/node_modules/busboy/lib/types/multipart.js:162:20)
librechat-api-75d85cd647-24hrp     at SBMH.ssCb [as _cb] (/app/node_modules/busboy/lib/types/multipart.js:394:37)
librechat-api-75d85cd647-24hrp     at feed (/app/node_modules/streamsearch/lib/sbmh.js:248:10)
librechat-api-75d85cd647-24hrp     at SBMH.push (/app/node_modules/streamsearch/lib/sbmh.js:104:16)
librechat-api-75d85cd647-24hrp     at Multipart._write (/app/node_modules/busboy/lib/types/multipart.js:567:19)
```

### Screenshots

Sorry my Ux is in French
![image](https://github.com/user-attachments/assets/e030b00b-eee2-4cf4-bbf0-f978fbbd5044)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

Edit : log format

**Comment by danny-avila** [comment]
Thanks for reporting this. I moved to troubleshooting because I thought there was something wrong with your config, but I see this isn't being properly handled. I'm already applying a patch

**Reply by danny-avila** [reply]
I'm addressing the fileConfig issues here: https://github.com/danny-avila/LibreChat/pull/4664

However, note that the RAG_API doesn't seem to work with legacy `.doc` files, at least every file I tried failed

**Reply by jogamned** [reply]
Awesome, 

thank you for your great work !
